ansible/roles/remove-old-pr-portal-apis/tasks/main.yml

```diff
@@ -0,0 +1,31 @@
+- name: list portals
+  uri:
+    url: "{{ portals_list_uri }}"
+    headers:
+      Authorization: "Bearer {{ APIGEE_ACCESS_TOKEN }}"
+    return_content: yes
+  register: portals
+
+- name: set portal_id
+  set_fact:
+    portal_id: "{{ portals.json.data[0].id }}"
+
+- name: get apidocs
+  uri:
+    url: "{{ portals_base_uri }}/{{ portal_id }}/apidocs"
+    method: GET
+    headers:
+      Authorization: "Bearer {{ APIGEE_ACCESS_TOKEN }}"
+    status_code: 200
+    return_content: yes
+  register: apidocs
+
+- name: set portal_apis
+  set_fact:
+    portal_apis: "{{ apidocs.json.data | to_json | from_json | json_query(_query) }}"
```

Comment:
Any particular reason for the `to_json | from_json`? (I don't do much Ansible so this might be an established pattern, just looks odd to me)

Reply:
Yeah, so the there is an issue with parsing the json directly using `from_json`. hence `to_json | from_json`